pavics_datacatalog/wps_processes/wps_pavicrawler.py

```diff
@@ -6,6 +6,10 @@
 
 from pavics import catalog
 
+#TODO: I suggest to move that configuration stuff into a config.py file which create an `env` variable that can be used elsewhere. Talk to Fred about the best way to go about it.
+#TODO: Change the name of the process. It not only crawls but also update the SOLR server.
+# How about making the THREDDS and SOLR servers explicit in the call, ie as inputs? I think better yet would be the crawling process and the update process.
+
```

Comment:
The Solr server should never be provided by the user, it has to be secured behind a strict firewall otherwise anyone can make modifications in it. Also, allowing specifying a thredds server would let users populate our Solr database with random external thredds servers... does not sound safe to me.